rest-api/alembic/versions/e26ea978c345_valuestring_to_text.py

```diff
@@ -0,0 +1,62 @@
+"""valuestring_to_text
+
+Revision ID: e26ea978c345
+Revises: 2be6f6d054e8
+Create Date: 2018-04-03 12:49:01.059482
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import model.utils
+from sqlalchemy.dialects import mysql
+
+from participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from participant_enums import WithdrawalStatus, SuspensionStatus
+from participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType
+from participant_enums import MetricSetType, MetricsKey
+from model.site_enums import SiteStatus, EnrollingStatus
+from model.code import CodeType
+
+# revision identifiers, used by Alembic.
+revision = 'e26ea978c345'
+down_revision = '2be6f6d054e8'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('participant_summary', 'last_modified',
+               existing_type=mysql.DATETIME(),
+               nullable=False)
```

Comment:
(Not only should we not need to change this column, we need to change measurement.value_string to be text...)

Reply:
@Seeker1911 it looks like we have a change to the schema that never got a migration made for it (making last_modified non-nullable) and so its being included in multiple PR migrations.  Two things:  first, its probably best to bust that migration out into its own PR, test & merge it, then merge master back into this and https://github.com/all-of-us/raw-data-repository/pull/656.  Second, you'll have to alter this migration to reflect the actual change b/c alembic can't pick it up (it has issues with some types of migrations).  You can look at https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/alembic/versions/c2dd2332a63f_questionnaire_response_value_string_.py for inspiration - it should be pretty similar.

Reply:
FWIW: I'm not sure we want to make last_modified not nullable. That migration when applied locally fails when you have existing participant_summary rows and strict mode is enabled. (It doesn't fail in Cloud SQL because strict mode isn't on, and so it defaults to zero... but still.)

We may just want to change lastModified to allow it to be nullable, even though in practice it will always be set moving forward.

Reply:
And indeed, you made that change explicitly here to make it nullable:

https://github.com/all-of-us/raw-data-repository/commit/1a30d2e027f2db896b45809beeafd12362827f52

I think the problem is this was done manually and the Python code was never updated to say it should be nullable. (Sorry for not catching that when I reviewed it.) Fix that, and the migration to make it not nullable will go away.